esmvaltool/cmorizers/data/formatters/datasets/esacci_sea_surface_salinity.py

```diff
@@ -0,0 +1,47 @@
+"""Common tools to CMORize NSIDC-0116 northern and sothern data."""
```

Comment:
Since this is still work in progress in #1832, I would not make the downloader and the formatter part of this PR. However, you may want to save the files (commits) for a future integration of this cmorizer.

Reply:
It is now merged

Reply:
And working fine in the new framework 👍 Could you please just fix the header and line 19 as well?